sdks/python/apache_beam/transforms/stats_test.py

```diff
@@ -154,13 +155,15 @@ def test_get_sample_size_from_est_error(self):
 
   @unittest.skipIf(sys.version_info < (3, 0, 0),
                    'Skip with py27 because hash function is not good enough.')
+  @retry(reraise=True, stop=stop_after_attempt(3))
```

Comment:
Could you please add a comment why these tests need to be retried? My understanding is that nondeterminism of hash() makes this test inherently flaky, therefore we retry.